src/lib.rs

```diff
@@ -35,7 +35,9 @@
 use std::time::Duration;
 
 use reqwest::{self, Client};
+use result::Result;
```

Comment:
Can you change this to a full path?

```suggestion
use crate::result::Result;
```